README.md

```diff
@@ -63,3 +62,14 @@ Check manifest:
 ```bash
 (venv) inv check-manifest
 ```
+
+To check compatibility with the pre-release version of InvenioRDM (invenio-app-rdm):
+
+```bash
+# Step 1 - install the pre-release dependencies
+(venv) pip install --pre -e .[dev_pre]
+# Step 2 - Run the pre-release tests
+(venv) inv test
+# if using uv run:
+uv run --extra dev_pre --prerelease=allow inv test
+```
```

Comment:
The workflow won't trigger until this is in the main branch, right? I don't see an Action that looks like it was run on the coded scheduled, Saturday. Or any github workflow Action entry that looks to check compatibility. So I guess this has to be run locally to test compatibility with v13?

Reply:
Right, it hasn't run. The way to test GH actions is really to merge them and run them.

In the past, I've investigated tools that purport to let you test GH actions locally (e.g. https://github.com/nektos/act) but they are beasts (the docker image for with all the environment was ~25GB) and I didn't quite get it to work in the end anyway.

Reply:
I see, that makes sense. Aside from the workflow, how about the check? Was `uv run --extra dev_pre --prerelease=allow inv test` run to check pre-release compatibility? What was the result of that run, if it was done locally?